src/overseer/heartbeat.clj

```diff
@@ -17,14 +16,15 @@
               [core :as core])))
 
 (defn start-heartbeat
-  "Start a process that will continually persist heartbeats via the DB"
-  [config conn current-job]
+  "Start a process that will continually persist heartbeats via the DB
+
+  current-job is an atom holding a map describing the current job,
+  i.e. the map just have {:job/id} of the currently running job"
```

Comment:
I think `at least` might be a good way to express the constraint here, but in general a job map should be a pretty well defined structure that could be referenced centrally